hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -46,10 +60,20 @@ const Dashboard = () => {
         }
     }, [dispatch, hardwareInOrders]);
 
+    const openEditTeamModal = () => {
+        //dispatch(getCurrentTeam());
+        dispatch(openTeamModalItem());
+    };
+
     return (
         <>
             <Header />
-            <ProductOverview showAddToCartButton={false} />
+            <ProductOverview showAddToCartButton />
```

Comment:
@Leo6Leo the props here are still wrong ^^

Reply:
<img width="949" alt="CleanShot 2022-06-17 at 09 59 03@2x" src="https://user-images.githubusercontent.com/36619969/174313117-e5e16bb3-c8f7-4429-bd3e-5a2b0a7666cd.png">

This is from develop branch, I didn't make any changes to this before. So What am I supposed to put here?